### DISCUSSION TITLE
How to show system tray items in Win10 taskbar style

### DISCUSSION THREAD

**Discussion by emufan** [valinet/ExplorerPatcher#2210] [Q&A] [answered]
I have enables Win10 taskbar style. Thx so much for making it possible!

But then, how would I be able to permanently show some item from the more-menu, as possible in Win10, so to always show e.g. icon from App x or icon from incoming mails or ...
Is there I menu to customize them or advanced setting to choose?

**Comment by Amrsatrio** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/9749af12-6936-43ef-a1a1-eb0c70a850d9)

And hit the checkbox at the bottom.

**Reply by emufan** [reply]
Unfortunately not working here. Because of this I opened this question. So please re-open. As it is not solved with this answer.

**Reply by Amrsatrio** [reply]
It works just fine on 22631.2361. What's your Windows version? Check `winver`.

**Comment by emufan** [comment]
22H2 (Build 22621.2283)

I can open this setting and change (as in Win10) to e.g.

![image](https://github.com/valinet/ExplorerPatcher/assets/22775515/1d959899-34fb-42a4-968e-14ac98793c23)

But After closing this setting page, it is set back to state before

![image](https://github.com/valinet/ExplorerPatcher/assets/22775515/e0bb35ba-2d16-4b04-98a4-3f238f0d947f)

So at least I cannot adjust there and perhaps this is the reason, why it is not taken into account.

**Reply by Amrsatrio** [reply]
Speaking about the dropdowns, It has always been like this since Windows 10. There are no fixes available for this.

Are you looking for this checkbox in this screen? Sorry if my Windows is not in English or your language, but at least you should get the idea.

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/11bd7f61-785f-4ac3-9906-cc72ff81ee5d)

It says "Always show all icons and notifications in the taskbar"

**Reply by emufan** [reply]
No, I don't want to display all (this would be the checkbox). I want to select which ones to display.

**Reply by pyrates999** [reply]
Did you try restarting explorer in the EP properties application after setting what system tray icons you want to show?

**Reply by emufan** [reply]
Yes, EP, Explorer, Windows, ...

The problem seems to be, that I cannot change anything in the dialog above. I can change but after leaving and reopening the dialog all settings are as before. So if I set "Symbol and notification" it returns to "only notification" automatically.

**Reply by pyrates999** [reply]
Did you click the OK button after making changes?

**Reply by emufan** [reply]
Yes, of course. 😉

Do you know where the settings are stored/which are perhaps the registry entries for this list? I would try to manipulate them there instead of the dialog, where I cannot change them.

**Reply by pyrates999** [reply, marked as answer]
Try the method here: https://www.makeuseof.com/system-tray-icons-windows-11/

**Reply by emufan** [reply]
Thanks a lot. The drag & drop feature described there is working. And is setting and storing eventually the option in the dialog!
